projects/pgai/db/sql/incremental/026-remove-destination-columns.sql

```diff
@@ -0,0 +1,54 @@
+do language plpgsql $block$
+declare
+    _vectorizer RECORD;
+    _target_schema text;
+    _target_table text;
+    _view_schema text;
+    _view_name text;
+    _config jsonb;
+begin
+    -- Loop through all vectorizers
+    for _vectorizer in select id, target_schema, target_table, view_schema, view_name, config from ai.vectorizer
+    loop
+        -- Extract the chunking config and chunk_column
+        _target_schema := _vectorizer.target_schema;
+        _target_table := _vectorizer.target_table;
+        _view_schema := _vectorizer.view_schema;
+        _view_name := _vectorizer.view_name;
+
+        -- Create new config:
+        -- Add destination config
+        _config := _vectorizer.config operator(pg_catalog.||) jsonb_build_object(
+            'destination', json_object(
+                'implementation': 'default',
+                'config_type': 'destination',
+                'target_schema': _target_schema,
+                'target_table': _target_table,
+                'view_schema': _view_schema,
+                'view_name': _view_name
+        ));
+
+        -- Update the vectorizer with new config
+        update ai.vectorizer 
+        set config = _config
+        where id operator(pg_catalog.=) _vectorizer.id;
```

Comment:
do we need to update the version of the config here?

Reply:
We do this in the 022 migration already which hasn't been released yet. So I don't think we have to do it here as well. That's honestly a bit of a mess... but it works.

I still feel like just not updating the versions and making the migration system simply work on the existence of keys instead of distinct versions, would be more robust. But I got voted out on that :D